nautobot/tenancy/api/serializers.py

```diff
@@ -4,35 +4,22 @@
 from nautobot.extras.api.serializers import NautobotModelSerializer, TaggedModelSerializerMixin
 from nautobot.tenancy.models import Tenant, TenantGroup
 
-# Not all of these variable(s) are not actually used anywhere in this file, but required for the
-# automagically replacing a Serializer with its corresponding NestedSerializer.
-from .nested_serializers import NestedTenantGroupSerializer, NestedTenantSerializer  # noqa: F401
```

Comment:
I think we *might* have some documentation somewhere about needing to import everything from nested_serializers into serializers? If so that should be revised/removed.

Reply:
I couldn't really find such documentation. Maybe the comments are the only thing that needs to be deleted?

Reply:
Think you're right - I can't find any such docs.